v2/concordium/service.proto

```diff
@@ -139,4 +139,12 @@ service Queries {
   // for API consistency.
   // The stream will end when all the non-finalized transaction hashes have been returned.
   rpc GetAccountNonFinalizedTransactions (AccountAddress) returns (stream TransactionHash);
+
+  // Suggest to a peer to connect to the submitted peer details.
+  // This, if successful, adds the peer to the list of given addresses.
+  rpc PeerConnect (PeerConnection) returns (BooleanResponse);
```

Comment:
What is the meaning of the boolean response? Can we do without it and signal errors using grpc errors?